docs/release-notes/master.adoc

```diff
@@ -13,11 +13,26 @@ include::topics/templates/document-attributes.adoc[]
 //Inclusive language statement
 include::topics/making-open-source-more-inclusive.adoc[]
 
+:!release-notes:
+
 [id="release-notes-introduction"]
 == Introduction
 {ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
 
-These release notes cover all _z_-stream releases of {ProductShortName} 6.1 with the most recent release listed first.
+These release notes cover all _z_-stream releases of {ProductShortName} 6.1 and 6.2, with the most recent release listed first.
```

Comment:
No it is release notes for 6.2. 
Think:
Release notes- 
- 6.2.x
- 6.1.x
- 6.0.x